src/components/GameView/GameOverlays.vue

```diff
@@ -235,7 +236,8 @@ export default {
 }
 .overlay-header {
     font-weight: bold;
-    background-color: #FFF4D7;
+    background-color: rgba(var(--v-theme-surface-2));
+    color: rgba(var(--v-theme-surface-1));
```

Comment:
Updated color reference to use theme and set the text to surface-1 (chocolate) from black